proposals/0264-stdlib-preview-package.md

```diff
@@ -160,4 +169,7 @@ A previous version of this proposal prescribed a window of time before promoting
 
 An alternative to keeping the semver major version at `0` is to not version the package at all. With this approach, the package could only be included by packages by specifying a branch-based dependency. However, this would be too restricting at this time, as a branch-based dependency can only be imported by other branch-based dependencies. This would effectively limit use of the `SwiftPreview` package to top-level applications only.
 
+### A Monolithic `SwiftPreview` Package
+
+An alternative to keeping the semver major version at `0` is to not version the package at all. With this approach, the package could only be included by packages by specifying a branch-based dependency. However, this would be too restricting at this time, as a branch-based dependency can only be imported by other branch-based dependencies. This would effectively limit use of the `SwiftPreview` package to top-level applications only.
```

Comment:
This paragraph is copied from the previous alternative and needs to be updated.

Reply:
Thanks, @anandabits!